src/main/java/com/direwolf20/buildinggadgets/common/building/modes/TargetedAxisChasingMode.java

```diff
@@ -0,0 +1,42 @@
+package com.direwolf20.buildinggadgets.common.building.modes;
+
+import com.direwolf20.buildinggadgets.common.BuildingGadgets;
+import com.direwolf20.buildinggadgets.common.building.IPlacementSequence;
+import com.direwolf20.buildinggadgets.common.building.IValidatorFactory;
+import com.direwolf20.buildinggadgets.common.building.placement.ExclusiveAxisChasing;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+
+/**
+ * Former "<i>Build to Me</i>".
+ * Logic is backed with {@link ExclusiveAxisChasing} where no attempt will be made at the ending (player) position.
+ * <p>
+ * This mode is designed for Building Gadget and does not guarantee to work with other gadgets.
+ */
+public class TargetedAxisChasingMode extends AtopSupportedMode {
```

Comment:
If the docBlock requires a former the wouldn't it have just made more sense to make it human readable? as in BuildToMeMode ?

Reply:
I agree on that localized name down't match code name is bad, and I will change it. However `TargetedAxisChasing` will not be changed since it is way more generic than the mode implementation.